test/graphql.createOrder.test.js

```diff
@@ -184,7 +184,7 @@ describe('createOrder', () => {
 
   it('creates a pending order if the collective is active and the payment method type is manual', async () => {
     const host = await models.Collective.create({
-      slug: 'host',
+      slug: 'host-collective',
```

Comment:
We cannot use `host` as a collective slug anymore in tests as it is a reserved keyword